userspace/libsinsp/threadinfo.cpp

```diff
@@ -360,6 +360,7 @@ void sinsp_threadinfo::init(scap_threadinfo* pi)
 
 	m_comm = pi->comm;
 	m_exe = pi->exe;
+	m_exepath = pi->exepath;
```

Comment:
I'm not sure that I fully understand `scap_read_proclist`, should I add the parsing code for `tinfo.exepath` in this function like `tinfo.exe`?
(Will `scap_dump` write all the event data to a file?)